jakartaee9/src/connectors_2_0.xsds

```diff
@@ -23,7 +23,7 @@
 	    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
 	    elementFormDefault="qualified"
 	    attributeFormDefault="unqualified"
-	    version="1.7">
+	    version="2.0">
```

Comment:
I can't comment on the rename, so...

I'm curious as to why this schema name became plural (connector -> connectors).
Yes, the name of the spec is plural, but this descriptor still defines a single connector, right?

Reply:
Yes, exactly, I changed it to plural because of the specification name. File named connector_2_0 didn't seem on par with the specification Connectors 2.0. 

But I am returning it back to singular as per your suggestion.